radar-upload-backend/src/main/java/org/radarbase/upload/dto/RecordMapperImpl.kt

```diff
@@ -46,7 +44,27 @@ class RecordMapperImpl: RecordMapper {
             modifiedDate = metadata.modifiedDate,
             committedDate = metadata.committedDate,
             logs = metadata.logs?.let {
-                LogsDto(url = "${uri.baseUri}/records/${metadata.id}/logs")
+                LogsDto(url = "${uri.baseUri}/records/${metadata.record.id}/logs")
```

Comment:
Note that the record ID and record metadata ID are equal via the one-to-one relationship. Excluding the `record` field may prevent a fetch.